en/services/ftp.md

```diff
@@ -454,3 +454,27 @@ The difference of the MAVLink implementation versus the standard are:
 - Missing final XOR out operation with `0xFFFFFFFF`.
 
 The effects of the initial value and final XOR operation are documented in this [brief tutorial on CRC computation of the Linux kernel](https://github.com/torvalds/linux/blob/master/Documentation/staging/crc32.rst).
+
+## MAVLink FTP URL Scheme
```

Comment:
> mavftp url format: QGC uses now this form: mftp://[;comp=<id>]<path> (so e.g. mftp://[;comp=1]version.json, where specifying the comp id is optional (mftp://version.json), and defaults to the current component. This allows e.g. to redirect an autopilot to download from a companion.

@bkueng Thanks. I updated this in MAVFTP here and also in component info doc. 

I don't really understand the "redirect" integration you talk about here. So can you please check what I have written here makes sense.

Reply:
Thanks @bkueng - I merged your comment on the redirect/companion use case. 

The autopilot supplies the URL with the location of the file. Did you make this a parameter with a default so it can be changed to the companion if needed?

This leads nicely into a question from the dev call - "where should component information files live". The concern is that a device might end up bricked if manufacturer-hosted component information disappeared. The proposal was that we recommend on the device, but that we mandate hosting defaults on github. Lorenz also mentioned perhaps allowing manufacturer to support their own version, with a default being the info on github. 

My thinking is that is we specify a default search path on the device which will be used if it exists, otherwise use location in a parameter that points to a default github file by default. Does that make sense?

Reply:
> Did you make this a parameter with a default so it can be changed to the companion if needed?

I don't have concrete plans to make use of it, but that could be an option.

Reply:
So you'll just have all the component information compressed on board PX4?